libs/ml/include/ml/ops/ops.hpp

```diff
@@ -170,7 +170,7 @@ class Ops
    * @return estimated charge amount, necessary for performing a backward pass on data of given
    * shapes.
    */
-  virtual OperationsCount ChargeBackward() const
+  virtual OperationsCount ChargeBackward()
```

Comment:
Are you sure about removing the const specifier? This method is intended to be a getter-only, e.g. a const one.